py_mmd_tools/nc_to_mmd.py

```diff
@@ -1373,7 +1373,9 @@ def to_mmd(self, collection=None, checksum_calculation=False, mmd_yaml=None,
         with open(self.output_file, 'w') as fh:
             fh.write(out_doc)
 
-    def get_data_access_dict(self, ncin, add_wms_data_access=False, add_http_data_access=True):
+    def get_data_access_dict(self, ncin, add_wms_data_access=False,
+                             custom_wms_link=None, custom_wms_layer_names=[],
```

Comment:
Simplify variable names. I also don't think variable should default to lists like this (memory issues or something).
```suggestion
                             wms_link=None, wms_layer_names=None,
```